packages/app-store/zohocalendar/lib/CalendarService.ts

```diff
@@ -266,8 +267,8 @@ export default class ZohoCalendarService implements Calendar {
         .filter((freebusy: FreeBusy) => freebusy.fbtype === "busy")
         .map((freebusy: FreeBusy) => ({
           // using dayjs utc plugin because by default, dayjs parses and displays in local time, which causes a mismatch
-          start: dayjs.utc(freebusy.startTime, "YYYYMMDD[T]HHmmss[Z]").toISOString(),
-          end: dayjs.utc(freebusy.endTime, "YYYYMMDD[T]HHmmss[Z]").toISOString(),
+          start: moment.utc(freebusy.startTime, "YYYYMMDDTHHmmssZ").toISOString(),
+          end: moment.utc(freebusy.endTime, "YYYYMMDDTHHmmssZ").toISOString(),
```

Comment:
@GAMHnile dayJS did not work, I kept getting this error "RangeError: Invalid time value", the reason was because it could not parse the date with the given format. I made an adjustment to make this work, but ran into another issue where ZOHO APIs was not returning a consistent format, for some time slots it did not not include time(just date) and this caused another issue. "moment" was able to handle it, so I just used "moment" to keep things simple and not account for difference scenarios "moment" probably has covered already.


Examples:
20241001T160000Z(With time)
20241005Z(Without time)